packages/cli/cli/src/commands/add-generator/addGeneratorToWorkspaces.ts

```diff
@@ -18,19 +18,14 @@ export async function addGeneratorToWorkspaces({
     groupName: string | undefined;
     cliContext: CliContext;
 }): Promise<void> {
+    if (generatorName === "docs") {
+        await addDocsToProject();
+        return;
+    }
```

Comment:
we should deprecate fern add docs and instead do `fern init --docs` and `fern init --api`